### DISCUSSION TITLE
[Question]: Plugin.loading status window

### DISCUSSION THREAD

**Discussion by VBrilyak** [danny-avila/LibreChat#5138] [Q&A]
### What is your question?

Hi all, 
Does anyone else have an issue with a plugin.loading tab?
Since v0.7.6-rc1 the status window never displays (this green/grey container). The plugins themselves and everything else works fine, except for that.
Kind regards,
Vitalii

### More Details
Before the v 0.7.6
<img width="647" alt="Screenshot 2024-12-30 at 11 29 37" src="https://github.com/user-attachments/assets/aa9b4e3b-bfe7-411b-95d2-59182ab82ea0" />
v0.7.6-rc1 and higher 
<img width="805" alt="Screenshot 2024-12-30 at 11 28 36" src="https://github.com/user-attachments/assets/0747c68b-c0bb-4926-a55e-610063c8925d" />

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Plugins are now deprecated and will no longer be supported. Please use Agents instead: https://librechat.ai/docs/features/agents

**Reply by VBrilyak** [reply]
I see, thanks!
